src/components/menu/side-nav.tsx

```diff
@@ -50,9 +50,10 @@ const MenuItem = ({id, url, title, children, activeTrail, level}: MenuItemProps)
     {
       // Non-active state.
       "font-normal text-stone-dark hocus:text-archway-dark hocus:font-medium": !activeTrail.includes(id) && activeTrail.at(-1) !== id,
-      "font-semibold text-stone-dark hocus:text-archway-dark hocus:font-medium hocus:before:content-[''] before:block before:w-[1px] before:h-full before:bg-stone-dark before:absolute before:left-0 before:top-0 before:scale-y-[1] before:transition": activeTrail.includes(id) && activeTrail.at(-1) !== id,
+      "font-semibold text-stone-dark hocus:text-archway-dark hocus:font-medium hocus:before:content-[''] before:block before:w-[2px] before:h-full before:bg-stone-dark before:absolute before:left-0 before:top-0 before:scale-y-[1] before:transition": activeTrail.includes(id),
+      "font-semibold text-stone-dark hocus:text-archway-dark hocus:font-medium hocus:before:content-[''] before:block before:w-[2px] before:h-full before:bg-fog before:absolute before:left-0 before:top-0 before:scale-y-[1] before:transition": activeTrail.includes(id) && activeTrail.at(-1) !== id,
```

Comment:
because `activeTrail.includes(id)` is in both lines, the second group only needs to be the classes that aren't included in the first bunch. Or you might want to do `&& activeTrail.at(-1) === id` on the first batch of classes. Either way, your call.

Reply:
After testing and double checking the mock, I realize I could just swap out the bg color instead. 🤔